### DISCUSSION TITLE
Сломалось удаление недавно игравших

### DISCUSSION THREAD

**Discussion by Lysenkov8** [Chimildic/YaMuTools#50] [Помощь с ошибкой] [answered]
Перестало работать удаление недавно игравших песен из плейлиста.  Проверил в Яндекс браузере и в Хроме. Выводит что нет треков для удаления.
Так же предложение может быть добавить в раздел "Вычитание треков" еще Историю как дополнительный плейлист, и там можно было бы и указывать сколько учитывать треков

**Comment by Chimildic** [comment, marked as answer]
Уточните версию расширения. Есть в настройках.

**Reply by Lysenkov8** [reply]
Версия 0.8.0

**Reply by Chimildic** [reply]
Обновите до 0.8.1. Пока недоступно для Firefox.

**Reply by Lysenkov8** [reply]
Получилось, спасибо большое

**Reply by kisoid** [reply]
> Обновите до 0.8.1. Пока недоступно для Firefox.

Ждём-с для FF

**Reply by Chimildic** [reply]
@kisoid добавлено

**Reply by kisoid** [reply]
> @kisoid добавлено

Проверил под Firefox. Всё работает, спасибо!
